samples/assistant/csharp-ooproc/AssistantSample/local.settings.json

```diff
@@ -3,10 +3,10 @@
   "Values": {
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
+    "CosmosDbConnectionString": "Local Cosmos DB emulator or Cosmos DB in Azure connection string",
```

Comment:
README file instructions to be updated for these settings.
https://github.com/Azure/azure-functions-openai-extension/blob/01e2d1b804a1af053a9982eeb5bf7d23766f6920/samples/assistant/README.md?plain=1#L79